docs/brand.md

```
@@ -0,0 +1,11 @@
+The project has a logo in svg format ([`logo.svg`](../brand/logo.svg)).
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)